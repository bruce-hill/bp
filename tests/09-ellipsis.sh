# The ellipsis matches text upto the following pattern, not counting newlines
# Example: bp '{"/*" .. "*/"}' matches "/* blah blah */" or "/**/"
bp '{"hello" .. "world"}'
