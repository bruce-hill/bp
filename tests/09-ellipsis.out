helloworld
hello;;;world
hello1234world
